jdi-light-mobile-tests/src/main/java/io/github/epam/sections/LoginForm.java

```diff
@@ -0,0 +1,10 @@
+package io.github.epam.sections;
+
+import com.epam.jdi.light.elements.composite.Form;
+import io.github.epam.entities.User;
+import org.openqa.selenium.WebElement;
+
+public class LoginForm extends Form<User> {
+	public WebElement name, password, loginButton;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6361799)